docs/resources/apigateway.md

```diff
@@ -16,7 +16,6 @@ An API gateway consists of the generic rules and configurations.
 ```
 resource "ionoscloud_apigateway" "example" {
     name              = "example-gateway"
-    logs              = true
```

Comment:
maybe we should leave `logs` in the example, just have them set to false?

Reply:
it's set to false by default anyway, so I don't think it would add anything to the example. I can add it if you think it helps.

Reply:
I don't have strong feelings about it, so let's just leave it as it is then